src/Promotions/ServiceProvider.php

```diff
@@ -62,12 +62,6 @@ private function bootPluginUpsells()
             Hooks::addAction('admin_enqueue_scripts', SaleBanners::class, 'loadScripts');
```

Comment:
We should prevent scripts from loading too

Reply:
Resolved: 44dad57ef8e69e392f2e6818e3a408382f463d43